index.html

```diff
@@ -203,10 +230,9 @@ <h3>Skills</h3>
                     <li class="linkedin-icon"><a href="#"><img src="img/linkedn-contact.png" alt="this is an icon for linkedin" class="media-icon"></a></li>
                     <li class="twitter-icon"><a href="#"><img src="img/twitter-contact.png" alt="this is an icon for twitter" class="media-icon"></a></li>
                 </ul>
-               <div class="bottom-line">
+               <div class="bottom-line mobile-v">
                    <img src="./img/bottom-line-contact.png" alt="A line that indicates bottom of the page">
                </div>
-
             </article>
         </section>
     </main>
```

Comment:
- [ ] Please, consider also fixing the `Get in touch` button. It should be left-aligned, considering the original design 👍 
<img width="1147" alt="Screen Shot 2022-06-02 at 11 56 59" src="https://user-images.githubusercontent.com/49064106/171583803-3213c003-6d27-4371-bcf1-2c15c54539fa.png">
> Figma design
<img width="615" alt="Screen Shot 2022-06-02 at 11 56 48" src="https://user-images.githubusercontent.com/49064106/171583843-4643fc7f-1c37-4b83-beae-b3387146cbb6.png">

Reply:
Implemented 👍